include/usb/usb_device.h

```diff
@@ -81,9 +81,24 @@ extern "C" {
  *  USB application interface
  **************************************************************************/
 
+struct usb_req_type_field {
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Comment:
In Bluetooth we do it the same way:
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/controller/ll_sw/pdu.h#L192

Reply:
I think it is perfectly fine to use bitfields in packed structures, provided the endian macros are used like in this example.